src/cli_common.c

```diff
@@ -371,3 +371,46 @@ void freeCliConnInfo(cliConnInfo connInfo){
     if (connInfo.auth) sdsfree(connInfo.auth);
     if (connInfo.user) sdsfree(connInfo.user);
 }
+
+/*
+ * Escape a string for JSON output(--json)
+ * Following characters should be escaped with '\' in JSON
```

Comment:
comment is outdated?

Reply:
I think these characters are still escaped with '\'

Reply:
yes, but that's not all that this function does.